src/utils/datadog.ts

```diff
@@ -0,0 +1,74 @@
+import { datadogRum } from '@datadog/browser-rum';
+
+/**
+ * Gets the configuration values for datadog based on the environment.
+ * It returns null if the environment is not staging or production.
+ * @param {string} environment - The environment to get the configuration values for.
+ * @example getConfigValues('staging');
+ * @returns {object|null} - The configuration values for datadog.
+ * **/
+const getConfigValues = (environment: string) => {
+    if (environment === 'production') {
+        return {
+            serviceName: 'deriv-bot',
```

Comment:
```suggestion
            serviceName: 'dbot',
```
to keep the consistency